addons/account/models/account_move.py

```diff
@@ -1505,7 +1505,7 @@ def _move_autocomplete_invoice_lines_values(self):
             # Shortcut to load the demo data.
             # Doing line.account_id triggers a default_get(['account_id']) that could returns a result.
             # A section / note must not have an account_id set.
-            if not line._cache.get('account_id') and not line.display_type:
+            if not line._cache.get('account_id') and not line.display_type and not line._origin:
```

Comment:
are you sure it will fix your issue? `line._origin` is always True right?

Reply:
@smetl Not when you are importing new line

Reply:
@smetl  line._origin is an empty record set when the line doesn't exist in db
So when the line already exists in db, it will not overwrite the account